src/main/webapp/dashboard-licenses.xhtml

```diff
@@ -20,37 +20,34 @@
         <f:viewAction action="#{dataverseHeaderFragment.addBreadcrumb(bundle['dashboard.license.header'])}"/>
     </f:metadata>
     <ui:composition template="/dataverse_template.xhtml">
-        <ui:param name="showDataverseHeader" value="false"/>
         <ui:define name="body">
 
+            <p:messages id="warningMessages" showDetail="true" closable="true" style="width: 80%">
+                <p:autoUpdate/>
+            </p:messages>
```

Comment:
Why not using standard messaging system used across whole dataverse?

Reply:
I tried using it while passing parameters but it just didn't work. I will try again after updating PR.

Reply:
Now I know why, the error messages in template are non-ajax based since they don't have p:autoupdate, so it willk never work here.